coordinator/server/client_api.go

```diff
@@ -39,6 +39,9 @@ type StatusResp struct {
 }
 
 type ManifestSignatureResp struct {
+	// The manifest signature - signed by the root ECDSA key. Changes upon each call due to a random component.
+	// example: MEYCIQCmkqOP0Jf1v5ZR0vUYNnMxmy8j9aYR3Zdemuz8EXNQ4gIhAMk6MCg00Rowilui/66tHrkETMmkPmOktMKXQqv6NmnN
+	ManifestSignatureRootECDSA []byte
```

Comment:
Is it possible to have this as a hex encoded string, similar to the `ManifestSignature` field?
Or would that negatively impact the verification workflow?

Reply:
I personally prefer and would recommend to stick with base64. While it is common to display a hash hex encoded and one might compare hashes just by looking at them, you can not verify a signature like that. Hence, base64 decoding the signature and piping it to a file sounds much more simpler and securer than having to juggle around with tools such as sed, awk or xxd. Also there is potential for confusion (big endian, little endian).